src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdkAppEngineDevServer2.java

```diff
@@ -90,6 +90,7 @@ public void run(RunConfiguration config) throws AppEngineException {
     arguments.addAll(DevAppServerArgs.get("clear_datastore", config.getClearDatastore()));
     arguments.addAll(DevAppServerArgs.get("datastore_path", config.getDatastorePath()));
     arguments.addAll(DevAppServerArgs.get("env_var", config.getEnvironment()));
+    arguments.addAll(DevAppServerArgs.get("application", config.getProject()));
```

Comment:
It's weird that we're changing from application to project here.

Reply:
Yea, `getApplication` didn't make much sense since it's too ambiguous of a term, so we chose `getProject` instead. Does it make more sense to just keep it the same as the parameter name `application`?

Reply:
The _Cloud SDK_ and Pantheon have moved to _project_; `application` has become an anachronism. But can I suggest `projectId`?  Pantheon uses _project_ as the project name, distinct from _Project ID_ (unique string) and _Project Number_.

Reply:
"project name", "project ID", and "project number" are three different things. Please be very clear about which this is. This has been a source of confusion and bugs in user apps before now.

Reply:
okay, well if we're changing anything now, it should probably directly map to what it is on cloud console, which I believe is `projectId`

Reply:
the devappserver is using the old appengine term for `projectId` which is `application`.

Reply:
also using projectId could solve this problem: https://github.com/GoogleCloudPlatform/app-gradle-plugin/issues/284

Reply:
`projectId` it is